### PR TITLE
not awful version of the powerfist pr

### PR DESCRIPTION
**Pull request by AdamElTablawy**
everyone said the powerfist using gas is bad, so i got rid of it

also, whoever coded the powerfist didn't understand that AP values on it do nothing as it directly applied damage

i changed it up some. now the powerfist, rather than using gas, changes its attack delay based on setting. on setting 1, it does 25 damage, on setting 2, it does 50 damage. attackspeed is adjusted so that depending on setting, it attacks slower (to compensate for the additional damage.) it also throws you the same distance universally and this only changes depending on the type.

the centurion's powerfist, the goliath, is adjusted so that it actually throws you farther now, as opposed to the literally nothing different it did before.

basically-setting 1, normal attackspeed, 4 hit crit. setting 2, 2hit crit, 2x the attackspeed.

everyone hated the gas stuff and this is easier to tweak, also means that it's now possible just to add a displacer glove etc or adjust the distance the powerfist throws you rather than being setting dependent

## Pre-Merge Checklist

- [x] You tested this on a local server.
- [x] This code did not runtime during testing.
- [x] You documented all of your changes.

:cl:
tweak: The powerfist no longer uses gas.
/:cl:

### CODE REVIEW

**Comment by buffyuwu** [comment]
proud parent

**Review by OneOneThreeEight — APPROVED** [review]
Probably the only melee weapon atm that feels really fucking good to use. You still get bodied by shotguns, though, but I think this is balanced being able to ignore all armor with a fist melee weapon that is otherwise very rare to find.
